proxy/http2/Http2Stream.cc

```diff
@@ -333,15 +333,16 @@ Http2Stream::do_io_close(int /* flags */)
     // by the time this is called from transaction_done.
     closed = true;
 
+    clear_timers();
+    clear_io_events();
+
+    bool stream_deleted = false;
```

Comment:
This can be moved inside the if() clause, right ?

Reply:
Yes, it can.  Will update.